api/client.go

```diff
@@ -287,6 +287,11 @@ func (c *Client) SetAddress(addr string) error {
 	return nil
 }
 
+// Address returns the URL address of the Vault client
```

Comment:
I'd say "set on the Vault client" because otherwise it sounds like the client is listening on some address :-)